edge/docs/installation/inception_internals.md

```diff
@@ -0,0 +1,122 @@
+|CME_LOG_LEVEL|Sets the vision-edge-cme container's logging level|
+|DLE_LOG_LEVEL|Sets the vision-edge-dle container's logging level|
+
+The valid logging levels are:
+|Setting|Description|
+|-|-|
+|TRACE|This is the most verbose logging level, probably only useful to developers working on the product or in extreme cases where troubleshooting at DEBUG level is not providing enough detail.
+|DEBUG|Output detailed debugging log messages. This should only be used when troubleshooting the system or when detailed logging is desired when performing development or system integration activity. This level will create very large log files quickly and should not be enabled for extensive periods in production. Setting log file rotation is highly recommended to avoid running out of disk space.
+|INFO|This is the default and provides a level of information suitable for a stable system
+|WARN|This level provides information that is of special significance, either non-fatal errors or events such as ancillary services being started or stopped. It is suitable for a stable production system.
+|ERROR|This level provides information when internal errors occur that may impact the functionality of the system.
+|PANIC|This level provides information when events that would normally cause a system crash occur. The MVIE containers implement panic recovery where possible, so the system processes should restart automatically but whenever events of this severity occur, they should be addressed.
+
+### Development Mode
+Setting `DEVMODE=true` in the `<install root>/volume/run/var/config/vision-edge.properties` will start the vision-edgecontroller container in development mode. When started in this mode:
+- Swagger documentation for the controller's REST API is enabled at https://`<host name>`/swagger/index.html
+- Additional configuration info is displayed in the controller logs (DEBUG logging level must also be enabled). Since the controller gets configuration information from (in order of decreasing precedence):
+  - environment variables set in the docker run command for the controller in the startedge.sh script
+  - the vision-edge.properties file
+  - the controller.json file
+  - default values
+
+  it can be useful to see it all in one place and to see which setting has taken precedence when the container is started.
+  
+  The complete set of configuration entries is output to the log file as a line of comma-separated values which indicate:
+
+  - variable name
+  - variable type 
+  - effective value 
+  - true if set in a configuration file
+  - true if setting equals the default value
+
+### CPU Mode
+By default, the vision-edge-dle container will not process inferences if there is no GPU on the edge node. Enabling CPU Mode will allow the DLE to run models on the CPU, which will run much slower than on GPU.
+ To enable CPU Mode, set `DLE_ENABLE_CPU_FALLBACK=TRUE` in the vision-edge.properties file.
```

Comment:
General comment about this file.  
There seems to be a lot of config file setups. 
I know that this is not the "real" documentation and I have not read those pubs.  How much of the information in here is required or even recommended? Have you considered a summary of default/recommended/required settings?

Reply:
> General comment about this file.
> There seems to be a lot of config file setups.
> I know that this is not the "real" documentation and I have not read those pubs. How much of the information in here is required or even recommended? Have you considered a summary of default/recommended/required settings?

None of this is required, and it's only recommended for people who want non-default behaviors. What's officially documented is how the product works out of the box.